vip-helpers/vip-stats.php

```diff
@@ -34,7 +34,7 @@ function wpcom_vip_top_posts_array( $num_days = 30, $limit = 10, $end_date = fal
 		
 		$args = array(
 			'days'  => $num_days,
-			'limit' => $limit,
+			'limit' => 100,
```

Comment:
I think an inline explanation for why $limit isn't used here would be valuable for the future.

Reply:
What do you think of this...
`'limit' => 100, // Due to caching, we request max limit and only return requested $limit below`

Reply:
I think that would be fine. Perhaps a reference back to this PR for fuller context.